tests/cpp/test_matmul.cpp

```diff
@@ -442,9 +445,13 @@ TEST_P(MatmulTestWithLayout, AmpereSwizzle) {
     NVF_CHECK(gdimx == expected_gdimx);
     NVF_CHECK(gdimy == expected_gdimy);
 
-    FusionProfiler::stop();
-    runtime = FusionProfiler::profile().kernel_time_ms;
-    ProfilerOptionsGuard::getCurOptions().unset(ProfilerOption::Enable);
+    if (!detectComputeSanitizer()) {
```

Comment:
Here, FusionProfiler is used for timing measurements, which doesn't mean anything with compute-sanitizer